playbooks/roles/airship-setup-deployer/tasks/helm-run.yml

```diff
@@ -1,63 +1,45 @@
 ---
 # Inspired by https://github.com/openstack/openstack-helm-infra/blob/master/roles/build-helm-packages/tasks/setup-helm-serve.yaml
-- block:
-    - name: checking if local helm server is running
-      wait_for:
-        delay: 5
-        timeout: 10
-        host: 127.0.0.1
-        port: 8879
-        state: started
-        msg: "Failure to connect on helm server -- Attempting automatic recovery"
-      register: _server_running
+- name: Copy systemd unit into place for helm server if it is not present
```

Comment:
can we somehow do an include here to include the helm-run.yml instead of duplicating the code?

```
- name: Include osh helm tasks
  include_role:
    name: deploy_osh
    tasks_from: helm-run
```

*Not tested, but theoretically it should work?

Reply:
In general yes, but I'm not sure if we should start with this place. More of the code is duplicated unfortunately